### PR TITLE
Move com.typesafe.sbt.pgp to com.jsuereth.sbt.gpp

### PR DESCRIPTION
**Pull request by timcharper**
It's odd for the namespace of the plugin to not match the artifact
namespace

Also, clean up obvious, low-hanging-fruit deprecation warnings

### CODE REVIEW

**Comment by timcharper** [comment]
`*** buffer overflow detected ***: /usr/lib/jvm/java-6-openjdk-amd64/bin/java terminated`

eek... I don't think I did that.

**Comment by timcharper** [comment]
@jsuereth any interest in this? or do you feel like the current namespace is ideal?

**Comment by ches** [comment]
A bit of a noisy diff, but I agree this is odd. Especially for something related to verifying authenticity, it'd be nice if provenance was crystal clear 😃  The artifact disparity in installation instructions already causes some pause, which I can't readily find any changelog to explain.

**Comment by timcharper** [comment]
noisy because I fixed all of the compiler warnings, which I should've noted.

**Comment by timcharper** [comment]
Oh, I did note. ^^ :)

**Comment by ches** [comment]
👍  I'd like to do some hacking with the plugin so if the namespace standardization is acceptable then it'd be nice to start from the basis of @timcharper's other cleanup.

**Comment by jsuereth** [comment]
yeah, I'm ok changing the namespace. This is one I wrote on my own, then adapted while still at typesafe.  Do we know why travis is failing?

**Comment by timcharper** [comment]
Unfortunately, no idea, but I heavily doubt that it's the fault of a change I made.

File: pgp-plugin/src/main/scala/com/jsuereth/sbt/SbtPgp.scala [hunk 1]
```diff
@@ -19,12 +19,11 @@ object SbtPgp extends AutoPlugin {
 
   override def trigger = allRequirements
   override def requires = sbt.plugins.InteractionServicePlugin && sbt.plugins.IvyPlugin
-  
-  // Note - workaround for issues in sbt 0.13.5 autoImport
-  object autoImportImpl {
+
+  object autoImport {
```
Comment (jsuereth) [inline pgp-plugin/src/main/scala/com/jsuereth/sbt/SbtPgp.scala]: wow, forgot I had added this before this bug was fixed :)

**Comment by jsuereth** [comment]
PR looks good. I kicked off the build again. Could be a bouncy-castle related issue....

**Comment by ches** [comment]
@timcharper Maybe just drop `sudo: false` into `.travis.yml` in a new commit on the branch and see if that fixes it?

**Comment by jsuereth** [comment]
@timcharper Would you mind adding `sudo:false` so I can merge thsi?  I'd like to take a crack at some extra fixes to sbt-pgp, and  anew release this weekend :)

**Comment by jsuereth** [comment]
Going to work on getting this in a bit.  Need a 1.1 release here...  Sorry for the LONG delay, you did great work, I don't mean to botch it up by being slow :)

**Comment by eed3si9n** [comment]
I kicked off Travis job and it's green now.

**Comment by eed3si9n** [comment]
Closing this in favor of https://github.com/sbt/sbt-pgp/pull/162 that rebases this to master.

**Pull request closed without merging**